### ISSUE TITLE
Problem when predicting sample data input.npz

### ISSUE THREAD

**Issue by QuangHieu1704** [microsoft/Swin3D#18]
Hi, I am working with your source code and faced a problem. With your sample data "input.npz", I got below results. This result like a mess. I don't know where am I wrong, so can you please share with me your code for making prediction on own point cloud? Thank you very much.

Note:
image1 is point cloud of "input.npz"
image2 is prediction
image3 is points of label "wall"
![image1](https://github.com/microsoft/Swin3D/assets/52438446/66f5c1e5-401d-4c0f-a4ff-56d4fe6d3840)
![image2](https://github.com/microsoft/Swin3D/assets/52438446/498103f0-26ef-45a2-81d2-36d1dfaca1e8)
![image3](https://github.com/microsoft/Swin3D/assets/52438446/a1d2bcca-9f15-451d-8f1f-ab93b8a03e8a)

**Comment by Yukichiii** [comment]
Do you mean the predictions output by `example.py`? The model in `example.py` is not loaded with weights, but rather randomly initialized, which naturally give result like a mess.

**Comment by QuangHieu1704** [comment]
Thanks for you reply. I have loaded the pretrained model. Actually, I didn't see the example.py script, I modified the examples/segmentation.py script.
![image](https://github.com/microsoft/Swin3D/assets/52438446/8300d545-1ad8-480d-a8c7-36e6fe6071e1)

**Comment by Yukichiii** [comment]
If you load the ckpt provided in this repo, it may also be a mess. This is because the pretrained model is pretrained on the Structured3D Dataset, which is a synthetic dataset. But the `input.npz` is a sample from ScanNet which is a real dataset. The model need to be finetuned on ScanNet to get reasonable segmentation result. Our finetuned model is provided in this [repo](https://github.com/Yukichiii/Swin3D_Task). If you want to forward and visualize the segmentation result, you can try this.
